README.md

```diff
@@ -1,3 +1,77 @@
-# dra-fair-teaching
+# Reproducible & FAIR Teaching Materials
 
-[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
\ No newline at end of file
+[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
+
+## About
+
+Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn.
+Prepared for the Digital Research Academy train-the-trainer program.
+
+## Requirements
+
+- [Quarto](https://quarto.org/)
+- [renv](https://rstudio.github.io/renv/articles/renv.html)
+
+<details>
+  <summary>Click here for details about `renv`</summary>
+```R
+# Bootstrapping renv 1.0.0 ---------------------------------------------------
+- Downloading renv ... OK
+- Installing renv  ... OK
+
+- Project '~/dra-fair-teaching' loaded. [renv 1.0.0]
+- None of the packages recorded in the lockfile are installed.
+- Using `renv::restore()` to restore the project library.
+Do you want to proceed? [Y/n]: Y
+
+The following package(s) will be updated:
+
+# CRAN -----------------------------------------------------------------------
+- base64enc     [* -> 0.1-3]
+- cachem        [* -> 1.0.8]
+- cli           [* -> 3.6.1]
+- digest        [* -> 0.6.33]
+- ellipsis      [* -> 0.3.2]
+- evaluate      [* -> 0.21]
+- fastmap       [* -> 1.1.1]
+- glue          [* -> 1.6.2]
+- highr         [* -> 0.10]
+- jquerylib     [* -> 0.1.4]
+- jsonlite      [* -> 1.8.7]
+- knitr         [* -> 1.43]
+- lifecycle     [* -> 1.0.3]
+- magrittr      [* -> 2.0.3]
+- memoise       [* -> 2.0.1]
+- mime          [* -> 0.12]
+- R6            [* -> 2.5.1]
+- rappdirs      [* -> 0.3.3]
+- rlang         [* -> 1.1.1]
+- stringi       [* -> 1.7.12]
+- stringr       [* -> 1.5.0]
+- vctrs         [* -> 0.6.3]
+- yaml          [* -> 2.3.7]
+
+# RSPM -----------------------------------------------------------------------
+- bslib         [* -> 0.5.1]
+- fontawesome   [* -> 0.5.2]
+- fs            [* -> 1.6.3]
+- htmltools     [* -> 0.5.6]
+- rmarkdown     [* -> 2.24]
+- sass          [* -> 0.4.7]
+- tinytex       [* -> 0.46]
+- xfun          [* -> 0.40]
+
+# Installing packages --------------------------------------------------------
+
+```
+</details>
+
+## License
+
+Creative Commons Attribution 4.0 International ([CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)).
+For details, see the [`LICENSE.md`](LICENSE.md) file.
+
+## Contributors
```

Comment:
Maybe a citation placeholder since we don't have that in place yet?

Reply:
added this to a new issue, see #14